go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/containerd/console v0.0.0-20170925154832-84eeaae905fa // indirect
 	github.com/containerd/containerd v1.0.2 // indirect
 	github.com/containerd/typeurl v0.0.0-20190228175220-2a93cfde8c20 // indirect
-	github.com/containernetworking/cni v0.6.0
+	github.com/containernetworking/cni v0.7.0-rc2
```

Comment:
is there a non-rc tag we can use? v0.7.0?

Reply:
I checked today, 0.7.0 is released. So I'll update patch set soon.